bin/run_hmis_system_tests.sh

```diff
@@ -0,0 +1,73 @@
+
+#!/bin/bash
+
+# usage
+# BRANCH_NAME=186406279-hmis-system-e2e-tests bin/run_hmis_system_tests.sh
```

Comment:
The fancy live-reloading stuff in the dev environment seems to break this (websockets I think is the issue). I didn't spend a lot of time trying to get it working but it seems like a bit of a rabbit hole

Reply:
Okay gotcha. If websockets are the issue, I wonder if it would work with running it locally with `yarn build` and `yarn preview`. We can go ahead and merge this though and figure that out later!

Reply:
Right, yeah that's the approach I have below down on line 49. I think maybe I misunderstood your question, you're wondering about running against a local repo rather than against a live server running in dev mode.

It should be easy to adjust this script to use a local repo rather than pulling from github. However in my environment I use different containers for the HMIS and the warehouse and there isn't a shared FS. I bet we could figure out a way to share a volume though.